test/hotspot/jtreg/runtime/logging/ExceptionsTest.java

```diff
@@ -45,9 +45,14 @@ static void updateEnvironment(ProcessBuilder pb, String environmentVariable, Str
 
     static void analyzeOutputOn(ProcessBuilder pb) throws Exception {
         OutputAnalyzer output = new OutputAnalyzer(pb.start());
+        System.out.println(output.getStdout());
```

Comment:
Debugging code?

If you really want to always print the output then the more common pattern is to use `out.reportDiagnosticSummary()` after all the checks have been done and so the test has passed (failing tests will print it anyway).

Reply:
The output is printed only if the failure happens in `OutputAnalyzer::shouldMatch()`, etc. I've updated the test so failures can happen other ways, so I cannot rely on this anymore.

Reply:
Sorry I don't follow. I'm saying there is no need to print stdout here because upon failure it will be printed anyway. If you want to see stdout even in passing cases then do this after all the `shouldXXX` checks.

Reply:
I fixed the test case to do all checks with `shouldXXX()`, and removed the stdout printing.